tests/plugins/a11yhelp/manual/commands.md

```diff
@@ -0,0 +1,52 @@
+@bender-tags: find, tc, 4.7.1, 16980
```

Comment:
Name of the test is misleading. It's purpose is to ensure that paste as plain text command is described, it's not a general comands test, so name like `plaintext.md` is more suitable.